web/src/components/ThemeContainer.tsx

```diff
@@ -79,6 +79,16 @@ const createTheme = (
           },
         },
       },
+      MuiAvatar: {
+        styleOverrides: {
+          root: {
+            backgroundColor: 'transparent',
+            '.MuiAvatar-fallback': {
+              display: 'none',
+            },
+          },
+        },
+      },
```

Comment:
🤔 Probably we are actually abusing the mui avatar for list item icons, there is also a `ListItemIcon` component, could you perhaps try to use that instead? No need though as we did the same before.
https://mui.com/material-ui/api/list-item-icon/

Reply:
If the icon is missing it will look like a missing profile image 😅. so I just hide it on fallback.
<img width="505" height="186" alt="image" src="https://github.com/user-attachments/assets/9d63f850-1201-4fcf-adb0-4f0c8a45eea7" />

Reply:
The difference is that for `ListItemIcon` you don't need to override the default of displaying a fallback profile image which is definitely not the "right" thing to do 😅  The `ListItemAvatar` is intended for profile pictures which is not something we use. Therefore please use `ListItemIcon`. We should also adjust all other usages of `ListItemAvatar` but we can do that separately.